### PR TITLE
feat: add an option to set a custom kustomize binary

### PR DESCRIPTION
**Pull request by alessiodionisi**
Following https://github.com/helmfile/chartify/pull/62.

- Added `--kustomize-binary` (`-k`) flag and `kustomizeBinary` config option to set a custom kustomize binary.

### CODE REVIEW

**Comment by yxxhero** [comment]
@alessiodionisi https://github.com/helmfile/chartify/releases/tag/v0.16.0  please use the latest chartify version.

**Comment by alessiodionisi** [comment]
> @alessiodionisi https://github.com/helmfile/chartify/releases/tag/v0.16.0 please use the latest chartify version.

Thank you for the quick release! I'll complete the PR today

**Comment by yxxhero** [comment]
@alessiodionisi please fix ci issue.

**Comment by alessiodionisi** [comment]
@yxxhero all green ✅

**Comment by yxxhero** [comment]
@alessiodionisi  did you test it locally in your project?

File: docs/index.md [hunk 1]
```diff
@@ -565,7 +568,7 @@ Flags:
       --skip-deps                         skip running "helm repo update" and "helm dependency build"
       --state-values-file stringArray     specify state values in a YAML file. Used to override .Values within the helmfile template (not values template).
       --state-values-set stringArray      set state values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2). Used to override .Values within the helmfile template (not values template).
-      --strip-args-values-on-exit-error   On exit error, strip the values of the args
+      --strip-args-values-on-exit-error   Strip the potential secret values of the helm command args contained in a helmfile error message (default true)
```
Comment (yxxhero) [inline docs/index.md:571]: please update the description in the go file.

**Comment by alessiodionisi** [comment]
> @alessiodionisi did you test it locally in your project?

yes!

File: docs/index.md (hunk 1 above)
Comment (alessiodionisi) [reply docs/index.md:571]: The output above was generated by executing `helmfile --help`

**Review by yxxhero — APPROVED** [review]

**Pull request merged**